content/programming-cookbook/tips-and-tricks/json-to-csharp-classes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: JSON to C# Classes
+showMetadata: true
+editable: true
+showToc: true
+---
+
+# Visual Studio
+
+- Example JSON data
```

Comment:
I suggest using small heading instead of bullet for explain that this is example of JSON data

For example:
```md
## Example JSON data
```